layouts/common.vue

```diff
@@ -30,6 +30,11 @@ export default {
   computed: {
     ...mapGetters({ isMobile: 'general/general/isMobile' })
   },
+  methods: {
+    onMobileMenuClicked (value) {
```

Comment:
If clicked stands for collapsed state and active, it should be named accordingly. I think `onMobileMenuToggle` is a good name

Reply:
In my logic collapsed stands for non active. You are right it's wrong I'll change names. Thanks :+1: